plugins/woocommerce/includes/rest-api/Controllers/Version2/class-wc-rest-system-status-v2-controller.php

```diff
@@ -1044,16 +1044,11 @@ public function get_active_plugins() {
 				return array();
 			}
 
-			$active_plugins = (array) get_option( 'active_plugins', array() );
-			if ( is_multisite() ) {
-				$network_activated_plugins = array_keys( get_site_option( 'active_sitewide_plugins', array() ) );
-				$active_plugins            = array_merge( $active_plugins, $network_activated_plugins );
-			}
-
-			$active_plugins_data = array();
+			$active_valid_plugins = wc_get_container()->get( PluginUtil::class )->get_active_and_valid_plugins();
```

Comment:
We were including network-activated plugins here too, but `get_active_and_valid_plugins()` doesn't. Should we also create a wrapper method for `wp_get_active_network_plugins()`?

Reply:
🤔 Oh hmm, I thought this method was still including network-activated plugins, but I didn't test. Thank you for doing so! I'll take another look.

Reply:
🤔 I'm trying to think of a case where we _wouldn't_ want to include active network plugins, and I'm drawing a blank. What if we change the helper function introduced in this PR to combine `get_active_and_valid_plugins` and `wp_get_active_network_plugins` into one list? That would probably actually be better for the usage in `woocommerce_is_active_in_current_site` anyway...

Reply:
Ok, in 37c871b0ac1779282ccc47e71da1b1b8beae8c6c I updated the method to combine `wp_get_active_and_valid_plugins` and `wp_get_active_network_plugins`.

Reply:
> > I'm trying to think of a case where we wouldn't want to include active network plugins, and I'm drawing a blank.

Yep. I agree. If, for any reason, that time comes, we can maybe supplement the function with an arg to filter results.